### PR TITLE
Add new Management Webhooks

### PR DESCRIPTION
**Pull request by michaelpaul**
**Description**

Adding new events:

**WEBHOOK** [paymentMethod.requestRemoved](https://docs.adyen.com/api-explorer/ManagementNotification/3/post/paymentMethod.requestRemoved)
Payment method request removed
**WEBHOOK** [paymentMethod.requestScheduledForRemoval](https://docs.adyen.com/api-explorer/ManagementNotification/3/post/paymentMethod.requestScheduledForRemoval)
Payment method request scheduled for removal

### CODE REVIEW

**Review by jillingk — DISMISSED** [review]

File: src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php [hunk 1]
```diff
@@ -19,15 +19,15 @@
 use Adyen\Model\ManagementWebhooks\ObjectSerializer;
 
 /**
- * NotificationDataMessage Class Doc Comment
+ * PaymentMethodRequestRemovedNotificationRequest Class Doc Comment
```
Comment (Aleffio) [inline src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php:22]: do we need to update the managementwebhook parser?

Comment (michaelpaul) [reply src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php:22]: I don't think we have one, just webhook models.

Comment (Aleffio) [reply src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php:22]: https://github.com/Adyen/adyen-php-api-library/blob/develop/src/Adyen/Service/ManagementWebhookParser.php#L10 this one?

Comment (michaelpaul) [reply src/Adyen/Model/ManagementWebhooks/PaymentMethodRequestRemovedNotificationRequest.php:22]: Good catch, file updated!

**Review by Aleffio — APPROVED** [review]

**Review by jillingk — APPROVED** [review]

**Pull request merged**